src/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -101,16 +112,20 @@ public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this
 						options.ValidateOnBuild = true;
 						options.ValidateScopes = true;
 					})
-					.ConfigureLogging(builder => builder.AddOmexLogging())
 					.Build();
 
+#pragma warning disable OMEX188 // InitializationLogger using OmexLogger is obsolete. DiagnosticId = "OMEX188"
 				InitializationLogger.LogInitializationSucceed(serviceNameForLogging);
+#pragma warning restore OMEX188 // InitializationLogger using OmexLogger is obsolete. DiagnosticId = "OMEX188"
```

Comment:
Where are the equivalent log events for the OpenTelemetry based logger?

Reply:
OpenTelemetry is not (yet) supported in this public repo.

Reply:
Okay, the old logging approach shouldn't really be removed then. The current approach is making the solution in this repo incomplete.

Reply:
Looks like logging components are coupled with other concerns. It would be better to split them. OTel configuration should not be integrated in this repo, consumers should be free to choose their own telemetry implementation.

Reply:
This can be addressed in another PR for rearranging components and librarys